hash.c

```diff
@@ -6035,7 +6039,7 @@ env_has_value(VALUE dmy, VALUE obj)
 
 /*
  * call-seq:
- *   ENV.rassoc(value)
+ *   ENV.rassoc(value) -> [value, name] or nil
```

Comment:
The order is reverse.

Reply:
Right you are.  (Reversed too much!)  Fixing.